File: storydoc-backend/src/main/java/io/storydoc/server/timeline/app/TimeLineDTO.java
package io.storydoc.server.timeline.app;


import io.storydoc.server.timeline.domain.TimeLineId;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class TimeLineDTO {
    private String name;
    private TimeLineId timeLineId;
    private List<TimeLineItemDTO> items;

}

File: storydoc-frontend/src/app/api/models/time-line-dto.ts
/* tslint:disable */
/* eslint-disable */
import { TimeLineId } from './time-line-id';
import { TimeLineItemDto } from './time-line-item-dto';
export interface TimeLineDto {
  items?: Array<TimeLineItemDto>;
  name?: string;
  timeLineId?: TimeLineId;
}
